Readme.md

```diff
@@ -0,0 +1 @@
+SSMT
```

Comment:
Please provide a brief description of what this PR is about, why is it necessary, and how to test this PR.

Reply:
I updated readme.md and also added few steps to test this PR